### DISCUSSION TITLE
Increase map size, or fullscreen map

### DISCUSSION THREAD

**Discussion by LostSoulfly** [teslamate-org/teslamate#1631] [Ideas]
It would be really nice to be able to show the map larger on the dashboard. I've only got one car, though, so I know this would impact those with two or more. An alternative would be to allow fullscreen viewing.

Perhaps a configuration variable in the settings for map size could be a compromise, to allow users to choose.

**Comment by hakong** [comment]
+1 on this. Would be nice to have a fullscreen map!
